client/src/components/StreetView.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { withGoogleMap, StreetViewPanorama } from 'react-google-maps';
+import Browser from '../util/browser';
+
+export type StreetViewAddr = {
+  bbl: string,
+  lat: number,
+  lng: number
+};
```

Comment:
As we continue to migrate parts of the codebase to TypeScript, we'll have a bunch of different modules with types like these, which, I suspect, are in reality the _same_ type passed throughout the whole codebase.  But because I didn't write the original code and I don't know what these types definitively contain, I'm just "subtyping" only the parts that individual components definitively need right now... once we've migrated more code to TS, we'll have a better understanding of what the "real" types are and can eventually coalesce them as needed.

Reply:
ok got it